python/servo/package_commands.py

```diff
@@ -287,43 +287,42 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
             msi_path = path.join(dir_to_msi, "Servo.msi")
             print("Packaged Servo into {}".format(msi_path))
         else:
-            dir_to_package = '/'.join(binary_path.split('/')[:-1])
-            dir_to_root = '/'.join(binary_path.split('/')[:-3])
-            resources_dir = dir_to_package + '/resources'
-            if os.path.exists(resources_dir):
-                delete(resources_dir)
-            shutil.copytree(dir_to_root + '/resources', resources_dir)
+            dir_to_temp = path.join(path.dirname(binary_path)) + "/targz"
+            resources_dir = dir_to_temp + '/resources'
             browserhtml_path = find_dep_path_newest('browserhtml', binary_path)
             if browserhtml_path is None:
                 print("Could not find browserhtml package; perhaps you haven't built Servo.")
                 return 1
-            print("Deleting unused files")
-            keep = ['servo', 'resources', 'build']
-            for f in os.listdir(dir_to_package + '/'):
-                if f not in keep:
-                    delete(dir_to_package + '/' + f)
-            for f in os.listdir(dir_to_package + '/build/'):
-                if 'browserhtml' not in f:
-                    delete(dir_to_package + '/build/' + f)
+            if os.path.exists(dir_to_temp):
+                print("Cleaning up from previous packaging")
+                delete(dir_to_temp)
+            print("Copying files")
+            shutil.copytree(path.join(self.get_top_dir(), 'resources'), resources_dir)
+            shutil.copytree(browserhtml_path, path.join(dir_to_temp + "/build"))
+            shutil.copytree(path.join(dir_to_temp[:-5], 'servo'), dir_to_temp)
```

Comment:
I think may have missed my [previous comment](https://github.com/servo/servo/pull/12850#discussion_r78609350), but `binary_path` already has the path to the servo binary. Also, looking at the [docs for the `shutil` module](https://docs.python.org/2/library/shutil.html), you'll see that `shutil.copytree` is for copying directories recursively, not files. Since the servo binary is just one file, another method from `shutil` would be more appropriate here.